test/extended/fixtures/forcepull-false-s2i.json

```diff
@@ -0,0 +1,36 @@
+{
+    "kind": "BuildConfig",
+    "apiVersion": "v1",
+    "metadata": {
+        "name": "ruby-sample-build",
+        "creationTimestamp": null,
+        "labels": {
+            "name": "ruby-sample-build"
+        }
+    },
+    "spec": {
+        "triggers": [],
+        "source": {
+            "type": "Git",
+            "git": {
+		"uri": "https://github.com/openshift/ruby-hello-world.git"
+            }
+        },
+        "strategy": {
+            "type": "Source",
+            "sourceStrategy": {
+		"from": {
+		    "kind": "DockerImage",
+		    "name": "docker.io/openshift/ruby-20-centos7"
+		},
+		"forcePull": false
+            }
+        },
+        "output": {
+            "to": {
+		"kind": "ImageStreamTag",
+		"name": "origin-ruby-sample:latest"
+            }
+        }
```

Comment:
outputs are actually optional so you could delete all the outputs and get rid of your image-stream.json file too.